docker-compose.yaml

```
@@ -52,10 +52,12 @@ services:
       - transmission-config:/config
       - ./transmission-settings.sh:/custom-cont-init.d/transmission-settings.sh:ro
 
+    # Note: ulimits may fail in rootless Docker if your user's ulimit is lower
+    # If you get errors, comment out or adjust these values based on `ulimit -n`
     ulimits:
       nofile:
-        soft: 32768
-        hard: 65536  # Increases allowed open files (important for high-speed torrenting)
+        soft: 4096   # Conservative default that works with rootless Docker
+        hard: 8192   # Reduced from 65536 for rootless compatibility
```

```suggestion
        soft: 1024   # Seemingly standard soft limit.
        hard: 4096   # Hard limit based on RPi4 running Alpine Linux as its host.
```

Checking against both my desktop and Pi, `ulimit -n -S` is both 1024 (though `ulimit -n -H` is 4096 on the pi and 524288 on my desktop. These values need to go down a lot, but I don't know if that is shared across containers in rootless Docker or not (i.e. if this should be lower still to allow other containers to have access to files without things shitting the bed)